core-feature-pack/src/main/resources/content/domain/configuration/host-master.xml

```diff
@@ -5,6 +5,9 @@
    and does not itself directly control any servers.
 -->
 <host name="master" xmlns="urn:jboss:domain:3.0">
+    <extensions>
+        <extension module="org.jboss.as.jmx"/>
```

Comment:
I know, and we still keep adding new stuff, there was a PR already for removing jmx subsystem from core. @bstansberry at the time didn't want to merge it because of some bigger refactoring needed in core that included jmx code. 
I don't mind you add this, it will just mean more work when we do move code back to full repo.